rumqttc/src/eventloop.rs

```diff
@@ -232,13 +234,48 @@ async fn connect(options: &MqttOptions) -> Result<(Network, Incoming), Connectio
     Ok((network, packet))
 }
 
+pub(crate) async fn get_tcp_stream(
+    addrs: String,
+    network_options: &NetworkOptions,
+) -> io::Result<TcpStream> {
+    let addrs = lookup_host(addrs).await?;
```

Comment:
Add timeout to `lookup_host`

Reply:
from its description: https://docs.rs/tokio/1.23.0/tokio/net/fn.lookup_host.html, it doesn't look like it would requires a timeout. should i still add it?